### ISSUE TITLE
frontend support for user to train pretrained models for image data and allow for them to build custom image classifiers

### ISSUE THREAD

**Issue by karkir0003** [DSGT-DLP/Deep-Learning-Playground#150]
**Describe the solution you'd like**
Frontend portion of image classification

Similar in look to homepage, but in image classification, user should be able to upload zipped image data or enter url to the zip
User should be able to drag and drop the preprocessing transforms to be applied on train and validation sets
User should be able to drag and drop the model architecture for their image classifier

Layers to be added: Conv2d, MaxPool2d, AdaptiveAvgPool2d, Dropout, BatchNorm2d + layers available in existing home page
(any other image classification related layers that are important can be added)
